lib/addons/p5.sound.js

```diff
@@ -10618,7 +10618,7 @@ var soundRecorder_ac = main.audiocontext;
  *      // send result to soundFile
  *      recorder.stop();
  *
- *      text('Done! Tap to play and download', width/2, height/2, width - 20);
```

Comment:
We probably shouldn't edit p5.sound.js here, as it comes from builds in https://github.com/processing/p5.js-sound that we periodically pull from. If we want to make a change to it, we'd have to make it in that repo for it to not get overwritten in the next build.